SA/design_concept/content/design/design_iteration_2_decisions_application.typ

```diff
@@ -2,13 +2,9 @@
 #import "@preview/tablex:0.0.6": tablex, hlinex
 #let load-drawio-svg(path, ..args) = image.decode(read(path).replace("Text is not SVG - cannot display", ""), ..args)
 
-= Application <application-decision>
+= Function Application <application-decision>
 
-The design suggested after the second iteration, as described in
-@design-proposal-2, leaves the application style open as a decision.
-This section makes that decision and elaborates on the reasons.
```

Comment:
Why would you like to remove this paragraph? There reference to this section is still there

Reply:
ah, I see, you removed them as well.

Wouldn't we need to add another paragraph instead to give more context?

Reply:
I think its ok. Like this, the sum type destruction and function application sections are just part of the concept description.

Reply:
ok 👍